src/options/optionsMain.ts

```diff
@@ -47,6 +47,7 @@ export async function getOptions(rawOptions: any): Promise<AppOptions> {
       },
     },
     nativefier: {
+      accessibilityPrompt: true,
```

Comment:
Okay, I like that you left it in the model as explicitly constantly true; if someone absolutely needs to disable it they can tweak the JSON manually, and we'll easily re-introduce the option that I got you to remove if user reports make us realize it was actually strongly required for legit use cases. 👍.